core/unix/signal.c

```diff
@@ -6172,7 +6205,7 @@ terminate_via_kill(dcontext_t *dcontext)
      * this may not exit all threads in the address space
      */
     block_cleanup_and_terminate(
-        dcontext, SYS_kill,
+        dcontext, IF_MACOS(SYSCALL_NUM_MARKER_UNIX +) SYS_kill,
```

Comment:
This is 64-bit Mac only (though 32-bit support should probably be dropped)

Reply:
Done.